frontend/src/components/messages/MessagesTab.vue

```diff
@@ -91,4 +91,9 @@ export default {
 .messages-button {
   display: inline-block;
 }
+
+.messages-button:focus {
```

Comment:
Hi Jen, could we add this style for Notification's buttons as well so they will look consistent?

Reply:
done :)